charts/celeborn/templates/_helpers.tpl

```diff
@@ -47,25 +47,25 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Selector labels
+*/}}
+{{- define "celeborn.selectorLabels" -}}
+app.kubernetes.io/name: {{ .Chart.Name }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
 {{/*
 Common labels
 */}}
 {{- define "celeborn.labels" -}}
-helm.sh/chart: {{ include "celeborn.chart" . }}
+helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
 {{ include "celeborn.selectorLabels" . }}
-{{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
+app.kubernetes.io/tag: {{ .Values.image.tag }}
```

Comment:
Tag can't be placed in common labels as it would prevent upgrade process.

Reply:
You are right, we should move this to selector label, ref to https://github.com/apache/incubator-celeborn/pull/1374#discussion_r1144671190